secretmanager/src/test/java/com/example/QuickstartIT.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.secretmanager.v1beta1.DeleteSecretRequest;
+import com.google.cloud.secretmanager.v1beta1.SecretManagerServiceClient;
+import com.google.cloud.secretmanager.v1beta1.SecretName;
+import com.google.common.base.Strings;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.UUID;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Integration (system) tests for {@link Quickstart}. */
+@RunWith(JUnit4.class)
+public class QuickstartIT {
+
+  private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+  private static final String SECRET_ID = "java-quickstart-" + UUID.randomUUID().toString();
+
+  @BeforeClass
+  public static void beforeAll() throws Exception {
+    Assert.assertFalse("missing GOOGLE_CLOUD_PROJECT", Strings.isNullOrEmpty(PROJECT_ID));
+  }
+
+  @AfterClass
+  public static void afterAll() throws Exception {
+    // Avoid running cleanup if project id was not given. This prevents an
+    // obtuse error from appearing in test output when the user forgets to set
+    // GOOGLE_CLOUD_PROJECT.
+    if (Strings.isNullOrEmpty(PROJECT_ID)) {
```

Comment:
Looks like we are asserting this to be false above - should prevent this scenario from happening?

Reply:
afterAll seems to always run, even if beforeAll fails.